src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": ">=7.1.3",
         "ext-xml": "*",
         "symfony/cache": "^4.4|^5.0",
-        "symfony/config": "^4.3.4|^5.0",
+        "symfony/config": "^4.4.11|^5.0.11",
```

Comment:
I've bumped this constraint because it need the fix from #37526 for the low-deps build.

Reply:
This should be `^4.4.11|~5.0.11|^5.1.3` actually. 🤔